docs/markdown/Getting Started/getting-started/installation.md

```diff
@@ -3,30 +3,24 @@ title: "Installing Pants"
 slug: "installation"
 hidden: false
 createdAt: "2020-02-21T17:44:53.022Z"
-updatedAt: "2022-05-03T20:48:35.453Z"
+updatedAt: "2022-07-12T00:00:00.000Z"
 ---
-Creating the launch script
---------------------------
 
-Pants is invoked via a launch script named `./pants` , saved at the root of the repository. This script will install Pants and handle upgrades.
+Most installations can use our one-step setup script to get a minimal version of Pants set up. If you need to make use of an advanced or non-standard setup, see our [manual installation](doc:manual-installation) page.
 
-First, set up a minimal `pants.toml` config file to instruct the script to download the latest 2.13 release:
+Using our one-step setup script
+-------------------------------
 
-```bash
-printf '[GLOBAL]\npants_version = "2.13.0.dev5"\n' > pants.toml
-```
+Pants has a launch script that handles downloading, bootstrapping, and upgrading Pants, which you need to save at the root of your repository. Pants also needs a `pants.toml` file, where you will eventually add all of the configuration needed to run testing, linting, and formatting rules. For now, it just needs to specify the version of pants that you want to use. 
```

Comment:
Either way, Pants needs to be capitalized. And probably helpful to mention the name `./pants` for the script in the first sentence

Reply:
Maybe line break with the `pants.toml` idea, so this is all more skimmable.